composer.json

```diff
@@ -3,26 +3,23 @@
     "description": "Automatic multi-tenancy for your Laravel application.",
     "keywords": ["laravel", "multi-tenancy", "multi-database", "tenancy"],
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Samuel Štancl",
-            "email": "samuel.stancl@gmail.com"
-        }
-    ],
+    "authors": [{
+        "name": "Samuel Štancl",
+        "email": "samuel.stancl@gmail.com"
+    }],
     "require": {
         "ext-json": "*",
-        "illuminate/support": "^6.0|^7.0|^8.0",
-        "facade/ignition-contracts": "^1.0",
+        "illuminate/support": "^6.0|^7.0|^8.0|^9.0",
         "ramsey/uuid": "^3.7|^4.0",
-        "stancl/jobpipeline": "^1.0",
-        "stancl/virtualcolumn": "^1.0"
+        "stancl/jobpipeline": "^1.5",
+        "stancl/virtualcolumn": "^1.2"
     },
     "require-dev": {
         "vlucas/phpdotenv": "^3.3|^4.0|^5.0",
-        "laravel/framework": "^6.0|^7.0|^8.0",
-        "orchestra/testbench-browser-kit": "^4.0|^5.0|^6.0",
-        "league/flysystem-aws-s3-v3": "~1.0",
-        "doctrine/dbal": "^2.10",
+        "laravel/framework": "^6.0|^7.0|^8.0|^9.0",
+        "orchestra/testbench-browser-kit": "^7.0",
+        "league/flysystem-aws-s3-v3": "~3.0",
+        "doctrine/dbal": "^2.10|^3.1.4",
```

Comment:
Does L9 use dbal 3.x?

Reply:
Not idea how did I update that here, reverted it back to 2.10. Laravel does not require it, but if it does I think it takes the latest.